builtin/stash--helper.c

```diff
@@ -10,6 +10,8 @@
 #include "run-command.h"
```

Comment:
Personally, I would move this commit and its parent very close to the commit that converts the `show` subcommand. "Close" as in: directly afterwards. I think that would make for a better story to the reader.